### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by TigerWolf**
Update documentation to improve for new users.

Work in progress as some commands are not working.

### CODE REVIEW

**Comment by bart88** [comment]
This does progress this issue #1 more thanks @TigerWolf

File: common.sh [hunk 1]
```diff
@@ -19,34 +19,34 @@ function setup_docker_machine() {
 
         # Check if the docker machine exists already, create one if not.
         if [[ $? == 1 ]]; then
-            notice "No ${MACHINE} environment found"
+            echo "No ${MACHINE} environment found"
```
Comment (bart88) [inline common.sh]: I'm guessing the method notice is missing in the common.sh. notice is lot more friendlier than echo. I suggest that we pull in the code that makes it work. @singularo @pingers

Comment (TigerWolf) [inline common.sh]: Why pull in another dependency? What value does it provide? I couldn't find it on a quick googles.

Comment (bart88) [inline common.sh]: `function notice() {
    echo -e "${COL_GREEN}$1${COL_RESET}"
}`

Comment (bart88) [inline common.sh]: Not a dependancy but just a clean way to represent -> notices, warnings and errors with different colours. All this functionality is missing. I'll raise a separate issue

**Comment by TigerWolf** [comment]
Cleaned up my mess.

**Comment by bart88** [comment]
👍  Merging it

**Comment by bart88** [comment]
fixes #3

**Pull request merged**